README.md

```diff
@@ -184,12 +184,11 @@ getFinalPrice(500); // 850
 When used with any iterable, it acts as to "spread" it into individual elements:
 
 ```javascript
-function foo(x, y, z) {
-    console.log(x, y, z);
-}
-
-let arr = [1, 2, 3];
-foo(...arr); // 1 2 3
+let args = ['https://github.com/metagrover/ES6-for-humans/', {method: 'GET'}];
+fetch(...args)
```

Comment:
While I agree that the foo examples leave a lot to imagination, this example with fetch can get a bit tricky for the first timers to grasp.

It'd be best if we can do use a collection:

```js
const ingredients = ['bread', 'butter', 'jam'];
```

and use it in a function (say, `makeBreakfast`) - using the spread operator.

Reply:
great idea Deepak! 

Does something like this work?

```js
const makeToast = (breadType, topping1, topping2) => {
  return `I had ${breadType} toast with ${topping1} and ${topping2}`;
};

const ingredients = ['sourdough', 'avocado', 'kale'];
makeToast(...ingredients); 
// "I had sourdough toast with avocado and kale"
```